README.md

```diff
@@ -15,31 +15,36 @@ Certificate authentication requires the following files:
 * **username.key** cfr [HowTo Get client access](#howto-get-client-access) !! Your private key **must** remain **secret**,
 * **username.crt** is located in [cert/pki/issued](https://github.com/jenkins-infra/openvpn/tree/master/cert/pki/issued), once an administrator signs  your request and publish it.
 
-```text
-client
-remote vpn.jenkins.io 443
-ca "~/.cert/jenkins/ca.crt"
-cert "~/.cert/jenkins/username.crt"
-key "~/.cert/jenkins/username.key"
-auth-user-pass
-dev tun
-proto tcp
-nobind
-auth-nocache
-script-security 2
-persist-key
-persist-tun
-user nobody
-group nobody
-```
-
-**With network manager client, you must enable the option :**
+**With NetworkManager client, you must enable the option :**
 
 `Use this connection only for resources on its network`
 
 ### DNS Problems
 
-If you are having issues connecting to resources behind the VPN, but the VPN appears to be working correctly, check your DNS settings.  Some providers seem to filter out requests to the zone.  To test, try `dig release.ci.jenkins.io`.
+If you are having issues connecting to resources behind the VPN, but the VPN appears to be working correctly, check your DNS settings.  Some providers seem to filter out requests to the zone.  To test, try `dig release.ci.jenkins.io`, you should get something like this:
+<details><summary>dig output</summary>
+```
```

Comment:
````suggestion

<details><summary>dig output (click to expand)</summary>

```text
````
- Since it's a markdown source file, you need to add empty lines between blocks to avoid breaking the final rendering
- Specifies a default language (`text`) for the fenced code block as per [MD040](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md040---fenced-code-blocks-should-have-a-language-specified)